### ISSUE TITLE
Cannot use import statement outside a module

### ISSUE THREAD

**Issue by butaminas** [EddyVerbruggen/nativescript-plugin-firebase#1621]
Using nativescript-vue and can't make my app run after upgrading to latest version of nativescript.

```javascript
"nativescript": {
  "id": "xxx",
  "tns-ios": {
    "version": "6.5.1"
  },
  "tns-android": {
    "version": "6.5.1"
  }
},
```

App.js:

```javascript
import Vue from "nativescript-vue";
import App from "./components/App";

const firebase = require('nativescript-plugin-firebase')
const {LocalNotifications} = require("nativescript-local-notifications")


firebase.init({
    showNotifications: true,
    showNotificationsWhenInForeground: true,
    onMessageReceivedCallback: (message) => {
        console.log('[Firebase] onMessageReceivedCallback:', {message});
        LocalNotifications.schedule(
            [{
                id: 1,
                title: message.data.title,
                body: message.data.body,
                silhouetteIcon: 'res://ic_app_icon',
                thumbnail: "res://icon",
                forceShowWhenInForeground: true,
                notificationLed: true,
                channel: i18n.t('messages_about_orders')
            }])
            .catch(error => console.log("doSchedule error: " + error));
    }
}).then(
    function () {
        console.log("firebase.init done");
    },
    function (error) {
        console.log("firebase.init error: " + error);
    }
);
```

I run `tns run android` and right after I get `Webpack build done` in the console I get these messages and a crash:

```javascript
nativescript-plugin-firebase: /Users/butaminas/Local Sites/getfast.co.nz-courier-app/platforms/android/.pluginfirebaseinfo not found, forcing prepare!
nativescript-plugin-firebase: running release build or change in environment detected, forcing prepare!
Cannot use import statement outside a module
```

**Comment by EddyVerbruggen** [comment]
If you comment the Firebase stuff, does it then still crash with the same error?

**Comment by butaminas** [comment]
I can comment out my app.js completely I still get the same result.

**Comment by butaminas** [comment]
I figured it out. It was the problem with `hooks` folder in the project root directory.

I don't know why this happened but after removing the plugin (the hooks directory was not removed) I removed the hooks folder manually and then installed the plugin again - the problem disappeared.  
  
I don't know if this is a bug or not. Perhaps the hooks folder should be handled accordingly be the plugin to avoid this?
